.fcrepo_wrapper

```diff
@@ -0,0 +1,10 @@
+# Place any default configuration for fcrepo_wrapper here
+port: 8984
+enable_jms: false
+
+download_dir: tmp/fcrepo_wrapper
```

Comment:
The `port` and `enable_jms` seem good to include here. I want to set my own `download_dir` and not have it overridden by Hyrax. You cool with removing that?